enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -251,13 +255,13 @@ scrape_configs:
       - source_labels: [__meta_kubernetes_pod_host_ip]
         action: replace
         target_label: node_ip
-      - source_labels: [__meta_kubernetes_pod_label_pod_template_hash, __meta_kubernetes_pod_name]
+      - source_labels: [__meta_kubernetes_pod_label_pod_template_hash, __meta_kubernetes_pod_name, workload]
```

Comment:
Some explanation around this would be useful. It's not clear to me why we need to match on workload source label - since it won't be added yet.

Reply:
This is a guard to require "workload" not be set, as a consistent theme on these kinds of rules we want to preserve the original if there is one already.  It could come from the exporter (like Cinnamon is configured to add a workload label) or from an annotation (annotation: {workload: blahblah}) before it hits a relabeling rule.

Reply:
So users can override the value?

Reply:
yes!  this and monitor_type are intended to work if set by user.  to be determined if/when we leverage that, of course.

Reply:
I really wish we had these reqs captured somewhere. It will make the support user story really difficult if these things are not captured anywhere, I think.